File: 06.streaming-os/StreamWatch/ios/StreamWatchiOS/Sources/Network/Phase1Repository.swift
import Foundation

struct Phase1Repository {
    func homeRoute() -> String {
        ApiRoutes.home
    }

    func libraryRoute() -> String {
        ApiRoutes.library
    }

    func profileBootstrapRoute() -> String {
        ApiRoutes.profileBootstrap
    }

    func tvHandoffStartRoute() -> String {
        ApiRoutes.tvHandoffStart
    }
}

File: 06.streaming-os/StreamWatch/android/StreamWatchAndroid/app/src/main/java/com/lsam/streaming/streamwatch/network/Phase1Repository.java
package com.lsam.streaming.streamwatch.network;

public final class Phase1Repository {
    public String getHomeRoute() {
        return ApiRoutes.HOME;
    }

    public String getLibraryRoute() {
        return ApiRoutes.LIBRARY;
    }

    public String getProfileBootstrapRoute() {
        return ApiRoutes.PROFILE_BOOTSTRAP;
    }

    public String getTvHandoffStartRoute() {
        return ApiRoutes.TV_HANDOFF_START;
    }
}
